extensions/test/BedrockMEAITests/BedrockMEAITests.NetFramework.csproj

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.AI.Abstractions" Version="9.4.3-preview.1.25230.7" />
+    <PackageReference Include="Microsoft.Extensions.AI.Abstractions" Version="9.5.0-preview.1.25262.9" />
```

Comment:
Same `Version` issue as above.

Reply:
This one is fine, no? is not form the nuspec and was already uppercase.